sandbox/Generated/UnitGenerator/UnitGenerator.SourceGenerator/FileGenerate.A.Generated.cs

```diff
@@ -57,7 +51,7 @@ public override int GetHashCode()
 
         public override string ToString()
         {
-            return "A(" + value + ")";
+            return value.ToString();
```

Comment:
different ToString result?(I don't know which is better)

Reply:
If I generate again with the unmodified branch, I seem to get a diff of this file.

It looks like `value.ToString()` is probably the implementation currently in use. Is this correct?
https://github.com/Cysharp/UnitGenerator/blob/master/src/UnitGenerator/CodeTemplate.tt#L88

Reply:
I recall changing it to a straightforward ToString because having type information made it harder to read, and it also reduced the convenience of ToString.

So, this implementation is OK!